### PR TITLE
update shipment-provider-api

### PR DESCRIPTION
**Pull request by traci707**
# [DEVDOCS-2028](https://jira.bigcommerce.com/browse/DEVDOCS-2028)

## What changed?
Updated per IA Refactor requirements

### CODE REVIEW

File: docs/api-docs/store-management/shipping/shipping-provider-api.md [hunk 1]
```diff
@@ -2,24 +2,24 @@
...
 - [Resources](#resources)
 
 </div>
 
-Shipping service providers wishing to offer shipping services and rates to BigCommerce merchants and shoppers can implement BigCommerce Shipping Provider endpoints. Once the service is implemented and accepted into BigCommerce's shipping Carrier Registry, merchants will have access to enable and configure the service through their BigCommerce control panel. Once enabled on a store, BigCommerce will automatically retrieve the service options and rates via the provider's endpoints and display them to merchants in the store's control panel and to shoppers on the storefront.
+Shipping service providers wishing to offer shipping services and rates to BigCommerce merchants and shoppers can implement BigCommerce Shipping Provider endpoints. Once they implement and accept their service into BigCommerce's shipping Carrier Registry, merchants will have access to enable and configure the service through their BigCommerce control panel. Once enabled on a store, BigCommerce will automatically retrieve the service options and rates via the provider's endpoints and display them to merchants in the store's control panel and to shoppers on the storefront.
```
Comment (ektravel) [inline docs/api-docs/store-management/shipping/shipping-provider-api.md:22]: "via" should not be used per the Style Guide Word List.
```suggestion
Shipping service providers wishing to offer shipping services and rates to BigCommerce merchants and shoppers can implement BigCommerce Shipping Provider endpoints. Once they implement and accept their service into BigCommerce's shipping Carrier Registry, merchants will have access to enable and configure the service through their BigCommerce control panel. Once enabled on a store, BigCommerce will automatically retrieve the service options and rates using the provider's endpoints and display them to merchants in the store's control panel and to shoppers on the storefront.

```

File: docs/api-docs/store-management/shipping/shipping-provider-api.md [hunk 2]
```diff
@@ -47,31 +47,31 @@ Please include:
 * Email
 * Description of the app
 
-To get your app ID, create an app in [Developer Tools](https://devtools.bigcommerce.com/) and fill out the information on [Step 3 Technical](https://developer.bigcommerce.com/api-docs/partner/app-store-approval-requirements). In the URL the app will have a unique ID. This is what is sent in exchange for a carrier ID which can be used to test the app.
+To get your app ID, create an app in [Developer Tools](https://devtools.bigcommerce.com/), and fill out the information on [Step 3 Technical](https://developer.bigcommerce.com/api-docs/apps/guide/requirements#technical). In the URL, the app will have a unique ID. Send the unique ID in exchange for a carrier ID to test the app.
```
Comment (ektravel) [inline docs/api-docs/store-management/shipping/shipping-provider-api.md:50]: The link for Step 3 Technical takes you to App Store Approval Requirements.

File: docs/api-docs/store-management/shipping/shipping-provider-api.md [hunk 3]
```diff
@@ -47,31 +47,31 @@ Please include:
...
+BigCommerce will send and receive data using JSON. The request for rates will always be formatted using the [Base Rate Request Model](https://developer.bigcommerce.com/api-reference/store-management/shipping-provider-api/shipping-provider/requestshippingrates). The response for rates should be formatted using the [Carrier Quote Object](https://developer.bigcommerce.com/api-reference/store-management/shipping-provider-api/shipping-provider/requestshippingrates). Format the request to check for merchant app credentials as Check Connection Options Request Payload, and the response should be formatted using the Check Connection Options Response Payload.
```
Comment (ektravel) [inline docs/api-docs/store-management/shipping/shipping-provider-api.md:71]: The Base Rate Request Model link takes you to the Request Shipping Rates endpoint.

Comment (ektravel) [inline docs/api-docs/store-management/shipping/shipping-provider-api.md:71]:
```suggestion
BigCommerce will send and receive data using JSON. The request for rates will always be formatted using the [Base Rate Request Model](https://developer.bigcommerce.com/api-reference/store-management/shipping-provider-api/shipping-provider/requestshippingrates). The response for rates should be formatted using the [Carrier Quote Object](https://developer.bigcommerce.com/api-reference/store-management/shipping-provider-api/shipping-provider/requestshippingrates). Format the request to check for merchant app credentials as Check Connection Options request payload, and the response should be formatted using the Check Connection Options response payload.

```

File: docs/api-docs/store-management/shipping/shipping-provider-api.md [hunk 4]
```diff
@@ -106,7 +106,7 @@ During the app setup, if the Check Connection Options URL is configured for the
 }
 ```
 
-**Example Response Check Connection**
+**Example response check connection**
```
Comment (ektravel) [inline docs/api-docs/store-management/shipping/shipping-provider-api.md:109]:
```suggestion
Response:

```

File: docs/api-docs/store-management/shipping/shipping-provider-api.md [hunk 5]
```diff
@@ -106,7 +106,7 @@ During the app setup, if the Check Connection Options URL is configured for the
 }
 ```
 
-**Example Response Check Connection**
+**Example response check connection**
 `/POST https://developerserver.com/check_connection_options`
```
Comment (ektravel) [inline docs/api-docs/store-management/shipping/shipping-provider-api.md:110]: I think this `/POST` code can be removed.

File: docs/api-docs/store-management/shipping/shipping-provider-api.md [hunk 6]
```diff
@@ -87,15 +87,15 @@ Example:
...
 
-## Control Panel Installation Workflow
+## Control Panel installation workflow
 
-During the app setup, if the Check Connection Options URL is configured for the carrier, an attempt to connect the carrier via the Shipping Manager UI or the Connect Carrier API causes a request to be made to that URL with the provided options. The resource should respond indicating if the credentials are valid and should provide an explanation of what is wrong. If no such URL is configured, this check will be skipped and the credentials are assumed valid as long as they pass type checks.
+During the app setup, if you configure the Check Connection Options URL for the carrier, an attempt to connect the carrier via the Shipping Manager UI or the Connect Carrier API causes a request to be made to that URL with the provided options. The resource should respond by indicating if the credentials are valid and explain what is wrong. If you did not configure this URL, this check is not required and the credentials are assumed valid as long as they pass type checks.
 
-**Example Request Check Connection**
+**Example request check connection**
```
Comment (ektravel) [inline docs/api-docs/store-management/shipping/shipping-provider-api.md:99]:
```suggestion
To validate connection options, send a `POST` request to `http://<yourserver.com>/check_connection_options`:

```

File: docs/api-docs/store-management/shipping/shipping-provider-api.md [hunk 7]
```diff
@@ -87,15 +87,15 @@ Example:
...
 `/POST https://developerserver.com/check_connection_options`
```
Comment (ektravel) [inline docs/api-docs/store-management/shipping/shipping-provider-api.md:100]: I think we can remove this line of code.

File: docs/api-docs/store-management/shipping/shipping-provider-api.md (hunk 2 above)
Comment (traci707) [reply docs/api-docs/store-management/shipping/shipping-provider-api.md]: I changed the link. 
I think this link will work.
https://developer.bigcommerce.com/api-docs/apps/guide/publishing#add-technical-information

File: docs/api-docs/store-management/shipping/shipping-provider-api.md [hunk 8]
```diff
@@ -127,26 +127,26 @@ During the app setup, if the Check Connection Options URL is configured for the
...
+To set up a carrier using the API, first, connect it using the Connect Carrier API. Make a request containing the connection settings required by your carrier. The ID of the carrier is required. The carrier ID will be issued by BigCommerce when your carrier is registered. All connection fields are unique per carrier. If your carrier doesn’t require any connection settings, then this object can be left empty.
 
 <!--
 title: "Sample Request "
 subtitle: "POST https://developerserver.com/shipping/carrier/connection"
 lineNumbers: true
 -->
 
-**Example Request Carrier Connection**
+**Example request carrier connection**
```
Comment (ektravel) [inline docs/api-docs/store-management/shipping/shipping-provider-api.md:150]:
```suggestion
To create a carrier connection, send a `POST` request. For example:

```

File: docs/api-docs/store-management/shipping/shipping-provider-api.md [hunk 9]
```diff
@@ -165,7 +165,7 @@ subtitle: "POST https://developerserver.com/shipping/carrier/connection"
 lineNumbers: true
 -->
 
-**Example Request with Empty Object**
+**Example request with empty object**
 `/POST https://developerserver.com/shipping/carrier/connection`
```
Comment (ektravel) [inline docs/api-docs/store-management/shipping/shipping-provider-api.md:170]: I think we can remove this line of code.

File: docs/api-docs/store-management/shipping/shipping-provider-api.md [hunk 10]
```diff
@@ -127,26 +127,26 @@ During the app setup, if the Check Connection Options URL is configured for the
...
 
 <!--
 title: "Sample Request "
 subtitle: "POST https://developerserver.com/shipping/carrier/connection"
 lineNumbers: true
 -->
 
-**Example Request Carrier Connection**
+**Example request carrier connection**
 `/POST https://developerserver.com/shipping/carrier/connection`
```
Comment (ektravel) [inline docs/api-docs/store-management/shipping/shipping-provider-api.md:151]: I think we can remove this line of code or add a link to the [Create a Carrier Connection](https://developer.bigcommerce.com/api-reference/store-management/shipping-api/shipping-carrier/postshippingcarrierconnection) endpoint.

File: docs/api-docs/store-management/shipping/shipping-provider-api.md [hunk 11]
```diff
@@ -175,14 +175,14 @@ lineNumbers: true
...
-Once connected, it’s possible to create shipping methods for a connected carrier in any shipping zone. Shipping zones can be queried using the Shipping Zones resource. For any zone, a request can be made to the Shipping Methods resource using the zone ID from the Shipping Zones resource to create a new method for the connected carrier. The shipping carrier’s ID is required in the type field.
+Once connected, it’s possible to create shipping methods for a connected carrier in any shipping zone. You can query shipping zones using the Shipping Zones resource. For any zone, a request can be made to the Shipping Methods resource using the zone ID from the Shipping Zones resource to create a new method for the connected carrier. You are required to enter the shipping carrier’s ID in the type field.
 <!--
 title: "Sample Request"
 subtitle: "POST https://api.bigcommerce.com/stores/{store_hash}/v2/shipping/zones/{zone_id}/methods"
 lineNumbers: true
 -->
 
-**Example Request Shipping Method**
+**Example request shipping method**
```
Comment (ektravel) [inline docs/api-docs/store-management/shipping/shipping-provider-api.md:186]:
```suggestion
To create a shipping method, send a `POST` request. For example:

```

File: docs/api-docs/store-management/shipping/shipping-provider-api.md [hunk 12]
```diff
@@ -204,7 +204,7 @@ subtitle: "POST https://api.bigcommerce.com/stores/{store_hash}/v2/shipping/zone
 lineNumbers: true
 -->
 
-**Example Response Shipping Methods**
+**Example response shipping methods**
```
Comment (ektravel) [inline docs/api-docs/store-management/shipping/shipping-provider-api.md:208]:
```suggestion
Response:

```

File: docs/api-docs/store-management/shipping/shipping-provider-api.md [hunk 13]
```diff
@@ -223,16 +223,16 @@ lineNumbers: true
...
-Whenever shipping rates are required, BigCommerce checks its internal cache for valid entries. If a valid entry exists, it will be used and the shipping carrier will not be called. If a valid cache entry does not exist, a request is made to the Quote URL with details of the items to be shipped, the shipping origin, and shipping destination. If any connection settings or zone settings are configured, these are also included. The shipping carrier must then respond with zero or more Shipping Quotes.
+Whenever shipping rates are required, BigCommerce checks its internal cache for valid entries. BigCommerce uses valid entries and does not call the shipping carrier. If a valid cache entry does not exist, BigCommerce makes a request to the Quote URL with details of the items to be shipped, the shipping origin, and the shipping destination. If you configured any connection settings or zone settings, include these. The shipping carrier must then respond with zero or more shipping quotes.
 
 <!--
 title: "Sample Request"
 subtitle: "POST https://developerserver.com/rate"
 lineNumbers: true
 -->
-**Example Request Shipping Rates**
+**Example request shipping rates**
```
Comment (ektravel) [inline docs/api-docs/store-management/shipping/shipping-provider-api.md:236]:
```suggestion
To request shipping rates, send a `POST` request. For example: 

```

File: docs/api-docs/store-management/shipping/shipping-provider-api.md [hunk 14]
```diff
@@ -326,7 +326,7 @@ subtitle: "POST https://developerserver.com/rate"
 lineNumbers: true
 -->
 
-**Example Response Shipping Rates**
+**Example response shipping rates**
```
Comment (ektravel) [inline docs/api-docs/store-management/shipping/shipping-provider-api.md:330]:
```suggestion
Response:

```

File: docs/api-docs/store-management/shipping/shipping-provider-api.md [hunk 15]
```diff
@@ -410,47 +410,47 @@ lineNumbers: true
...
-To do so, the metafields must meet the following requirements:
+To pass metadata in a rate request, the metafields must meet the following requirements:
 
-* must be a product or variant metafields (category, brand, and other metafields are not passed in rate requests)
+* must be a product or variant metafields (you cannot pass category, brand, and other metafields in rate requests)
 * `permission_set` on the metafield must be `read` or `write`
 * `namespace` on the metafield must match this format: `shipping_carrier_carrier_id` (example: `shipping_carrier_72`)
 
-The `carrier_id` is provided during the carrier registration process described in the [Sign-up Section](https://developer.bigcommerce.com/api-docs/store-management/shipping/shipping-provider-api)
+The carrier registration process described in the [Sign-up Section](https://developer.bigcommerce.com/api-docs/store-management/shipping/shipping-provider-api#sign-up) provides the `carrier_id`.
```
Comment (ektravel) [inline docs/api-docs/store-management/shipping/shipping-provider-api.md:426]:
```suggestion
The carrier registration process described in the [Sign-up Section](https://developer.bigcommerce.com/api-docs/providers/shipping#sign-up) provides the `carrier_id`.

```

File: docs/api-docs/store-management/shipping/shipping-provider-api.md [hunk 16]
```diff
@@ -410,47 +410,47 @@ lineNumbers: true
...
 * [API Reference > Store Management > Catalog > Product Variant Metafields](https://developer.bigcommerce.com/api-reference/catalog/catalog-api/product-variants-metafields)
 
-## Submitting the App
+## Submitting the app
 
-Before submitting your app make sure you have the following information.
+Before submitting your app, make sure you have the following information.
 
-**Single Carrier vs Multi Carrier**
+**Single carrier vs. multi carrier**
```
Comment (ektravel) [inline docs/api-docs/store-management/shipping/shipping-provider-api.md:436]:
```suggestion
**Single-carrier vs. multi-carrier**

```

File: docs/api-docs/store-management/shipping/shipping-provider-api.md [hunk 17]
```diff
@@ -410,47 +410,47 @@ lineNumbers: true
...
 
-Whether the app is a single carrier or multiple carriers. A single carrier app will offer one service such as USPS. A multi carrier app will offer more than one carrier such as USPS, DHL and Canada Post.
+Whether the app is a single carrier or multiple carriers. A single carrier app will offer one service, such as USPS. A multi carrier app will offer more than one carrier such as USPS, DHL, and Canada Post.
```
Comment (ektravel) [inline docs/api-docs/store-management/shipping/shipping-provider-api.md:440]:
```suggestion
A single-carrier app will offer one service, such as USPS. A multi-carrier app will offer more than one carrier such as USPS, DHL, and Canada Post.

```

File: docs/api-docs/store-management/shipping/shipping-provider-api.md (hunk 10 above)
Comment (traci707) [reply docs/api-docs/store-management/shipping/shipping-provider-api.md:153]: I will remove the code.

File: docs/api-docs/store-management/shipping/shipping-provider-api.md (hunk 9 above)
Comment (traci707) [reply docs/api-docs/store-management/shipping/shipping-provider-api.md:172]: Yes, sounds good.

**Pull request merged**